src/context/Theme.tsx

```diff
@@ -348,6 +348,12 @@ export const alternateConnectorImageBackgroundSx: SxProps<Theme> = {
     background: (theme) => connectorCardLogoBackground[theme.palette.mode],
 };
 
+// This is the hex code for the monaco editor background in dark mode.
+export const shardTableRow = {
```

Comment:
Interesting. The following color object is used for the purpose identified in the comment above: `monacoEditorWidgetBackground`. Are these colors unsuitable for the shard information table?

**NOTE:** At the time of commenting, I have not looked at any screenshots or manually evaluated the code myself so I do not know what this application of color looks like. Given the comment, I am led to believe the aforementioned color object _should_ be applicable here.

Reply:
I removed this as it is for the details pages. To answer the question... I don't know. The comment was the one next to the `sx` prop that this styling was taken from.

I thought it looked fine though.